app-ios/Modules/Sources/Timetable/InfoLabel.swift

```diff
@@ -26,10 +26,10 @@ struct InfoLabel: View {
             .padding(.vertical, 4)
             .foregroundColor(labelColor)
             .background(backgroundColor)
-            .cornerRadius(4)
+            .cornerRadius(50)
             .overlay {
                 if let strokeColor = strokeColor {
-                    RoundedRectangle(cornerRadius: 4)
+                    RoundedRectangle(cornerRadius: 50)
```

Comment:
How about using [capsule](https://developer.apple.com/documentation/swiftui/capsule) ?
I think this is useful for this situation!

Reply:
Thank you, it looks much better 👍 

`Capsule` has the same look as `RoundedRectangle(cornerRadius: 50)`, but actually I could not understand why this one works well...
The tag component's height is 24 and its required radius is 50, which is much larger than the half size of its height.
Maybe, the cornerRadius of RoundedRectangle was not respected...? 

RoundedRectangle(cornerRadius: 50) | Capsule
:--:|:--:
<img src="https://github.com/DroidKaigi/conference-app-2023/assets/43767445/afb0e80b-aee9-4fa4-aacf-983486d1af1f" width="300"> | <img src="https://github.com/DroidKaigi/conference-app-2023/assets/43767445/3168c70e-d340-48f0-a88b-73d42b3c1536" width="300">